src/app/components/report-problem/report-problem.component.css

```
@@ -0,0 +1,45 @@
+}
+
+.full-width {
+  width: 100%;
+}
+
+.description {
+  margin-bottom: 16px;
+  color: var(--text-color);
+}
+
+::ng-deep .submit-btn {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)